### DISCUSSION TITLE
A Shareable freely edit link

### DISCUSSION THREAD

**Discussion by dsmrs** [outline/outline#5012] [Ideas]
Something I love in pads like [Etherpad](https://etherpad.org/) or [Hedgedoc](https://hedgedoc.org/) is the ability for anyone to edit a document without having any account. In HedgeDoc(HackMD or CodiMD) you can easily activate or deactivate that function.

![hedgedoc](https://user-images.githubusercontent.com/12203221/224076422-a767146a-2181-4fd8-b56a-f1e4e464ca7e.png)

As we are using *Outline* for our team documentation, sometime we use it to take collaborative notes of a meeting with externals (non-profit partners, external member, etc.). We would love to be able to share a link where people can edit without having to have an account to the doc and write on the document we shared. And relock it later on.

**Comment by kosssi** [comment]
:+1: It's same https://github.com/outline/outline/discussions/3897
